action.yml

```diff
@@ -87,12 +89,11 @@ runs:
           --cluster ${{ inputs.environment }} \
           --service ${{ inputs.ecr_repository }}-${{ inputs.environment }}
 
-    # https://docs.aws.amazon.com/cli/latest/reference/ecs/wait/services-stable.html
-    - name: Await service stability
+    - name: Wait for service stability
       shell: bash
       id: check-service-stability
       run: |
-        aws ecs wait services-stable \
+        poetry run --directory ${{ github.action_path }} actions_helper wait-for-service-stability \
```

Comment:
Should we use `working-directory: ${{ inputs.working_directory }}` instead of `--directory`. In this case it would be consistent with the other ones.